### DISCUSSION TITLE
Please document the `--var` and `--define` arguments

### DISCUSSION THREAD

**Discussion by AaronBeaudoin** [cloudflare/workers-sdk#1807] [Archive]
By running `wrangler dev -h` I found this:

```
--var        A key-value pair to be injected into the script as a variable  [array]
--define     A key-value pair to be substituted in the script  [array]
```

Some more explanation regarding these would be appreciated. A few points:

1. Neither of these arguments are anywhere at https://developers.cloudflare.com/workers/wrangler/commands.
2. When I tried using `--var`, it took me some time to figure out why `TEST=testing` didn't work. Turns out the `=`, which seemed like the natural character, actually needed to be a `:`. It seems this is currently only discoverable by trial and error.
3. It's not clear to me what `--define` does. Some docs with example code would go a long way towards clearing these up.

Thanks so much!

**Comment by rozenmd** [comment]
I've moved this discussion to https://github.com/cloudflare/cloudflare-docs/issues/6022
